src/app/compressed-air-assessment/results-panel/inventory-performance-profile/inventory-performance-profile.component.ts

```diff
@@ -304,8 +343,53 @@ export class InventoryPerformanceProfileComponent implements OnInit {
           unloadingData: unloadingData,
           controlType: this.selectedCompressor.compressorControls.controlType
         });
+      } 
+    }
+    return chartData;
+  }
+  getAvgOpPointsData(): Array<ProfileChartData> {
+    let profileSummary: Array<ProfileSummary> = new Array<ProfileSummary>();
+    this.compressedAirAssessment.compressedAirDayTypes.forEach(dayType => {
+      let profiles: Array<ProfileSummary> = this.compressedAirAssessmentResultsService.calculateBaselineDayTypeProfileSummary(this.compressedAirAssessment, dayType, this.settings);
```

Comment:
Can you just assign this to profileSummary?

Reply:
No, because only one day type average would be graphed.